index.html

```diff
@@ -110,6 +110,83 @@ <h3 class="my-works-project-title">Tonic</h3>
                 </section>
             </div>
         </div>
+    <!--
+    <script>
```

Comment:
- [ ] I kindly suggest that you remove commented code because it does not make your code look neat.